multiqc/utils/search_patterns.yaml

```diff
@@ -429,6 +429,8 @@ homer/FreqDistribution:
   fn: "petag.FreqDistribution_1000.txt"
 hops:
   fn: "heatmap_overview_Wevid.json"
+hostile:
+  contents: '"reads_in"'
```

Comment:
If it's a json file, `fn: "*.json"` can be additionally added.

It's also a good practice to set the `num_lines` to the minimum number of lines by which you expect to meet the `contents` line. It limits reading potentially huge JSON files and speeds up the file search. See, for example:
https://github.com/MultiQC/MultiQC/blob/main/multiqc/utils/search_patterns.yaml#L602-L605